### PR TITLE
OWNERS: update to use new labels

### PR DESCRIPTION
**Pull request by spiffxp**
This PR is primarily about using the new labels defined in https://github.com/kubernetes/test-infra/pull/23084 to help us better triage incoming PRs

But of course, all of the commits have scarily vague "update OWNERS" messages.  I added details in the commits to explain what else was going on, but I'll drop a review on this pointing out specifics

### CODE REVIEW

**Comment by spiffxp** [comment]
/hold
for comment

File: apps/cert-manager/OWNERS [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - munnerz
+- munnerz
```
Comment (spiffxp) [inline apps/cert-manager/OWNERS:4]: So, one change that happened a lot was trying to make formatting consistent:
- deindent list items
- newline between approver/reviewer/emeritus_approver lists (and put them in that order)

File: apps/k8s-io/OWNERS [hunk 2]
```diff
@@ -1,10 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-# deploying this is still a manual process, so restricting
-# to approvers who are comfortable doing so
-options:
-  no_parent_owners: true
-
```
Comment (spiffxp) [inline apps/k8s-io/OWNERS:7]: Now that we have jobs auto-deploy k8s.io changes on PR merge, this is no longer needed 🎉

File: apps/kubernetes-external-secrets/OWNERS [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-    - wg-k8s-infra-leads
+- ameukam
+- spiffxp
```
Comment (spiffxp) [inline apps/kubernetes-external-secrets/OWNERS:5]: AFAIK @ameukam and I are the only two who have domain expertise on wg-k8s-infra's use of this, I'd rather signal that to the other leads

File: apps/perfdash/OWNERS [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
+
 approvers:
-  - bartsmykla
-  - mm4tt
-  - wojtek-t
+- mm4tt
+- wojtek-t
+
+emeritus_approvers:
+- bartsmykla
```
Comment (spiffxp) [inline apps/perfdash/OWNERS:8]: AFAIK we haven't seen much approval from this contributor over the last while so it's time to emeritus

File: apps/prow/OWNERS [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
+
+reviewers:
+- ameukam
+- bentheelder
+- spiffxp
 
 labels:
-- area/prow/bump
 - sig/testing
+- area/apps/prow
```
Comment (spiffxp) [inline apps/prow/OWNERS:15]: Formatting change: for apps, reorder so the "owning group" label is first, and the "app-specific" label is second

File: apps/publishing-bot/OWNERS [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- dims
+- nikhita
+- sttts
```
Comment (spiffxp) [inline apps/publishing-bot/OWNERS:6]: Best guess based on kubernetes/publishing-bot/OWNERS

File: audit/OWNERS [hunk 7]
```diff
@@ -1,4 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+approvers:
+- ameukam
+- spiffxp
+- thockin
```
Comment (spiffxp) [inline audit/OWNERS:6]: AFAIK these are the only people who have reviewed the automated PRs here

File: groups/OWNERS [hunk 8]
```diff
@@ -14,11 +14,15 @@ options:
...
-  - dims
-  - thockin
-  - spiffxp
+- cblecker
+- dims
+- thockin
+- spiffxp
+
+reviewers:
+- nikhita
```
Comment (spiffxp) [inline groups/OWNERS:23]: @nikhita has helped get us on the path to delegated approval

File: groups/committee-code-of-conduct/OWNERS [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-    committee-code-of-conduct
+- committee-code-of-conduct
```
Comment (spiffxp) [inline groups/committee-code-of-conduct/OWNERS:4]: I'm not sure this OWNERS file ever worked, so I converted to list items to fix

File: hack/verify-conftest.sh [hunk 10]
```diff
@@ -42,8 +42,8 @@ function main() {
         infra/gcp/clusters/projects/*/*/resources
     )
     local conftest_flags=(
-        # override the default of looking for $(pwd)/policy
-        --policy "${REPO_ROOT}/policies/"
+        # be explicit about looking for $(pwd)/policy
+        --policy "${REPO_ROOT}/policy"
```
Comment (spiffxp) [inline hack/verify-conftest.sh:46]: To match the new `area/policy` label, I relocated `policies/*` to `policy/` which should just work by default, but I felt like staying explicit was better

File: infra/gcp/backup_tools/OWNERS [hunk 11]
```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- cip-approvers
+- wg-k8s-infra-leads
+reviewers:
+- cip-reviewers
+- release-engineering-approvers
+- release-engineering-reviewers
```
Comment (spiffxp) [inline infra/gcp/backup_tools/OWNERS:9]: I used `k8s.gcr.io/images/k8s-staging-artifact-promoter/OWNERS` as the basis for other CIP-related stuff

**Review by spiffxp — COMMENTED** [review]
Annotated specific changes that weren't just "changed/added labels"

**Comment by spiffxp** [comment]
/uncc @chases2 @hasheddan 
/cc @cblecker @nikhita 
sig-contribex owners
/cc @ameukam @dims 
wg-k8s-infra owners
/cc @listx @cpanato @puerco 
CIP / release-engineering owners

File: infra/gcp/backup_tools/OWNERS (hunk 11 above)
Comment (listx) [reply infra/gcp/backup_tools/OWNERS:9]: LGTM

Comment (justaugustus) [reply infra/gcp/backup_tools/OWNERS:9]: LGTM for CIP!

**Pull request merged**